prep-html-syntax/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <h1 style="color:rgb(2, 0, 128);">My favorite Star Wars character is Grogu!</h1>
+  </head>
```

Comment:
Be sure to only put meta information in the `<head>` of your document, such as `<title>`. Your `<h1>` should go in the `<body>`, since it's meant to be part of the visible page content.